### PR TITLE
Drop Python 3.8

### PR DESCRIPTION
**Pull request by SimonHeybrock**

### CODE REVIEW

File: conda/meta.yaml [hunk 1]
```diff
@@ -37,10 +37,8 @@ requirements:
     - python
     - tbb-devel {{ tbb_devel }}
   run:
-    - graphlib-backport # for python < 3.9
     - numpy
     - python
-    - pyyaml
```
Comment (jl-wynen) [inline conda/meta.yaml:43]: Why is this not needed any more?

**Review by jl-wynen — COMMENTED** [review]
Why were some notebooks reformatted?

**Comment by SimonHeybrock** [comment]
> Why were some notebooks reformatted?

Fix for new Plopp, it got formatted when saving. Might be pre-black?

File: conda/meta.yaml (hunk 1 above)
Comment (SimonHeybrock) [reply conda/meta.yaml:43]: I tried to find why it was there but could not. It might be from some ancient config?

**Comment by jl-wynen** [comment]
> > Why were some notebooks reformatted?
> 
> Fix for new Plopp, it got formatted when saving. Might be pre-black?

We format notebooks with black? I don't see any config for that.

**Review by jl-wynen — APPROVED** [review]

**Comment by nvaytet** [comment]
> We format notebooks with black? I don't see any config for that.

We generally format the notebooks, but we don't enforce in CI. This means that some old notebooks may still not be formatted.

**Pull request merged**